doc/source/whatsnew/v1.5.0.rst

```diff
@@ -732,6 +732,7 @@ Timedelta
 ^^^^^^^^^
 - Bug in :func:`astype_nansafe` astype("timedelta64[ns]") fails when np.nan is included (:issue:`45798`)
 - Bug in constructing a :class:`Timedelta` with a ``np.timedelta64`` object and a ``unit`` sometimes silently overflowing and returning incorrect results instead of raising ``OutOfBoundsTimedelta`` (:issue:`46827`)
+- Bug in constructing a :class:`Timedelta` from a large integer or float with ``unit="W"`` silently overflowing and returning incorrect results instead of raising ``OutOfBoundsTimedelta`` (:issue:`47268`)
```

Comment:
Could you also add a note in the `Other API Changes` section mentioning that various timstamp/timedelta constructors that would overflow now raise a  `OutOfBoundsTimedelta`/`OutOfBoundsTimestamp` instead of an `OverflowError` (they appear to be based off of different subclasses).

Should hopefully be a catchall for all these improvements you've been making

Reply:
good idea, updated